policy/auto-join.json

```diff
@@ -43,43 +43,75 @@
                         "UserAssigned",
                         "SystemAssigned"
                     ]
+                },
+                {
+                  "field": "Microsoft.ContainerService/managedClusters/nodeResourceGroup",
+                  "notLike": "MC_FL_*"
+                },
+                {
+                  "field": "name",
+                  "notEquals": "hub"
```

Comment:
can you add an "escape hatch" with a tag?
i.e: if the cluster has a tag "IgnoreFleetDINEPolicy: true" don't consider it.

Reply:
Done!